src/Pages/CalculatorQuestion.jsx

```diff
@@ -26,7 +26,7 @@ const CalculatorQuestion = ({ setUserResponse, userResponse }) => {
 
   const deliverQuestion = () => {
     const { text, tooltip, options } = Questions.filter(
-      (q) => q.id == number
+      (q) => q.id === number
```

Comment:
I think number is a string when we get it from useParams(). That's why we had the `==` instead of `===`. We'd need to parseInt() on this line to use `===`.

Reply:
got it! fixed!